.gitignore

```diff
@@ -6,3 +6,6 @@
 
 # Folders
 node_modules
+
+#stupid OSX trash
```

Comment:
This is as meaningful as I can make it :P i really dont know why, but Mac OSX adds this stupid file sporadically.